Blockzilla/Info.plist

```diff
@@ -67,6 +67,10 @@
 		<key>openUrl</key>
 		<string>org.mozilla.ios.Klar.openUrl</string>
 	</dict>
+	<key>NimbusAppChannel</key>
+	<string>developer</string>
+	<key>NimbusAppName</key>
+	<string>focus_ios</string>
```

Comment:
These were added by `PlistBuddy`, but I'm not sure if we should default them in code, or add them here.

The previous behavior was to error out of Nimbus construction, which isn't great now.